nym-wallet/src-tauri/src/state.rs

```diff
@@ -449,6 +449,30 @@ impl WalletStateInner {
     pub fn remove_validator_url(&mut self, url: config::ValidatorConfigEntry, network: Network) {
         self.config.remove_validator_url(url, network)
     }
+
+    pub fn calculate_coin_delta(
+        &self,
+        coin1: &DecCoin,
+        coin2: &DecCoin,
+    ) -> Result<DecCoin, BackendError> {
+        match coin1.amount.cmp(&coin2.amount) {
+            std::cmp::Ordering::Greater => {
+                let delta = DecCoin {
+                    amount: coin1.amount - coin2.amount,
+                    denom: coin1.denom.clone(),
+                };
+                Ok(delta)
+            }
+            std::cmp::Ordering::Less => {
+                let delta = DecCoin {
+                    amount: coin2.amount - coin1.amount,
+                    denom: coin1.denom.clone(),
+                };
+                Ok(delta)
+            }
+            std::cmp::Ordering::Equal => Ok(coin1.to_owned()),
+        }
+    }
```

Comment:
This could be potentially simplified:
Instead of coding a dedicated function to calculate the delta, I could simply calculate the sub then use the absolute value (one-liner). But `DecCoin` type does not implement any kind of `abs()`. But maybe we can implement that easily?